spec/app/ics-721-nft-transfer/README.md

```diff
@@ -35,29 +35,35 @@ The IBC handler interface & IBC routing module interface are as defined in [ICS
 
 ### Data Structures
 
-Only one packet data type is required: `NonFungibleTokenPacketData`, which specifies the class id, class uri, token id's, token uri's, sender address, and receiver address.
+Only one packet data type is required: `NonFungibleTokenPacketData`, which specifies the class id, class uri, class data, token id array, token uri array, token data array, sending address, and receiving address.
 
 ```typescript
 interface NonFungibleTokenPacketData {
   classId: string
   classUri: string
+  classData: byte[]
```

Comment:
how should this be serialized? if I have `classData` that is `AAA` does that serialize to `[65, 65, 65]`?

Reply:
Resolved in https://github.com/cosmos/ibc/pull/899/commits/a6a2b330fb0ca1d74cea6067c9c8a66d0f969238